orchagent/flexcounterorch.cpp

```diff
@@ -61,13 +62,22 @@ void FlexCounterOrch::doTask(Consumer &consumer)
                 const auto &field = fvField(valuePair);
                 const auto &value = fvValue(valuePair);
 
-                if (field ==  POLL_INTERVAL_FIELD)
+                if (field == POLL_INTERVAL_FIELD)
```

Comment:
I do not recall we need the FIELD suffix now, when did we introduce them?

Reply:
They were introduced before. I can have a separate PR to clean it up later if that is preferred